user-interface/src/main/java/life/qbic/datamanager/views/account/PersonalAccessTokenComponent.java

```diff
@@ -120,11 +120,14 @@ private void showCreatedPersonalAccessToken(String rawTokenText) {
     Icon copyIcon = VaadinIcon.COPY_O.create();
     copyIcon.addClassName(IconSize.SMALL);
     copyIcon.addClassName("clickable");
-    copyIcon.addClickListener(
-        event -> UI.getCurrent().getPage().executeJs("window.copyToClipboard($0)", rawTokenText));
+
     Span rawToken = new Span(rawTokenText);
     Span personalAccessTokenWithIcon = new Span(rawToken, copyIcon);
     personalAccessTokenWithIcon.addClassName("token-text");
+
+    copyIcon.addClickListener(
```

Comment:
NIT: why did you move this down? I think it makes more sense to keep the listener close to the icon for which it is added.